modules/clients/mount_wekafs.sh

```diff
@@ -91,8 +94,8 @@ function retry {
 
 mount_command="mount -t wekafs -o net=udp $backend_ip/$FILESYSTEM_NAME $MOUNT_POINT"
 if [[ ${mount_clients_dpdk} == true ]]; then
-  getNetStrForDpdk $(($NICS_NUM-1)) $(($NICS_NUM)) "$gateways" "$subnets"
-  mount_command="mount -t wekafs $net -o num_cores=1 -o mgmt_ip=$eth0 $backend_ip/$FILESYSTEM_NAME $MOUNT_POINT"
+  getNetStrForDpdk 1 $NICS_NUM "$gateways" "$subnets" "-o net="
```

Comment:
here you pass first gw then subnets and above function expects the other way

Reply:
right we are not using subnets, I will remove it